community_topics_workflow.md

```diff
@@ -0,0 +1,63 @@
+
+**Note:** If you have any ideas on how the Workflow can be improved, please create an issue in this repository or pull request against this document.
+
+## Workflow
+
+**Note:** This is a rough scenario and it can vary depending on a topic's complexity and other nuances, for example, when there is a mass agreement up-front.
+
+### Preparation stage
+
+- [ ] A Committee person checks the topic's content, asks the author / other persons to provide additional information if needed (adds the `waiting` label in this case).
+- [ ] Adds the topic to the [Board](https://github.com/orgs/ansible-community/projects/2/views/1).
+
+### Discussion stage
+
+- [ ] If the topic is ready to be discussed, the Committee person:
+
+  - [ ] Adds the `discussion_topic` label.
+  - [ ] Moves the topic to the `In Discussion` column on the [Board](https://github.com/orgs/ansible-community/projects/2/views/5).
+  - [ ] Open the discussion by adding a comment asking the Community and the Committee to take part in it.
+- [ ] No sync discussion is needed (there are no blockers, complications, confusion, or impasses).
+
+### Voting stage
+
+- [ ] Depending on the topic's complexity, after 1-2 weeks pass since the discussion was open, the Committee person formulates vote options based on the prior discussion and gives participants reasonable amount of time to propose changes to the options (no longer than a week). The person summarizes the options in a comment and also establishes a date when the vote begins if there are no objections about the options / vote date.
+- [ ] In the vote date, the vote starts with the comment of a Committee person which opens the vote and establishes a date when the vote ends ($CURRENT_DATE + no longer than 14 days).
+- [ ] The Committee person labels the topic with the `active-vote` label and moves the topic to the `Active Vote` column on the [Board](https://github.com/orgs/ansible-community/projects/2/views/5).
+- [ ] The Committee person adds `[Vote ends on $YYYY-MM-DD]` to the beginning of the topic's description.
+
+### Voting result stage
+
+- [ ] The next day after the last day of the vote, the Committee person:
+
+  - [ ] Removes the `active-vote` label.
+  - [ ] Add a comment that the vote ended.
+  - [ ] Changes the beginning of the topic's description to `[Vote ended]`.
+  - [ ] Counts the votes separating ones made by Community and made by Committee and creates a summary comment containing the result.
+  - [ ] Asks another Committee person to count the votes and provide the summary.
+- [ ] Up to the [next](https://github.com/ansible/community/blob/main/meetings/README.md#wednesdays) Community meeting, at least two Steering Committee members count the votes and summarize the result in comments.
```

Comment:
Is this step really needed? I think it would be better to remove formal needs for the meetings completely. We can still meet and discuss things, but we shouldn't have a process that depends on these meetings.

Reply:
Good catch, agreed, changed (I removed Up to the [next](https://github.com/ansible/community/blob/main/meetings/README.md#wednesdays) Community meeting in the first sentence and replaced it with Bullhorn in the second one), thank you!